rc_user

```diff
@@ -2,3 +2,4 @@ export USERNAME='$(shell whoami)'
 export API_URL=//mf-chsdi3.dev.bgdi.ch/$USERNAME
 export APACHE_BASE_PATH=/$USERNAME
 export E2E_TARGETURL=https://mf-geoadmin3.dev.bgdi.ch/$USERNAME
+export SHOP_URL=//shop-shopbgdi.openshift.puzzle.ch
```

Comment:
The url are already set in rc_file, the problem is for dev `shop-bgdi.dev.bgdi.ch`doesn't work. SHould we use //shop-shopbgdi.openshift.puzzle.ch as dev url?